features/step_definitions/setup_steps.rb

```diff
@@ -230,19 +237,69 @@ def create_buyer_brief (brief_name,framework_slug,lot,user_id)
   brief_data["briefs"]["frameworkSlug"] = framework_slug
   brief_data["briefs"]["lot"] = lot
   brief_data["briefs"]["userId"] = user_id
+  brief_data["briefs"]["startDate"] = '31/12/2016'
+  brief_data["briefs"]["specialistRole"] = 'developer'
+  brief_data["briefs"]["organisation"] = 'Driver and Vehicle Licensing Agency'
+  brief_data["briefs"]["importantDates"] = 'Yesterday'
+  brief_data["briefs"]["evaluationType"] = ['pitch']
+  brief_data["briefs"]["contractLength"] = '1 day'
+  brief_data["briefs"]["backgroundInformation"] = 'Make a flappy bird clone except where the bird drives very safely'
+  brief_data["briefs"]["essentialRequirements"] = ['Can you do coding?']
+  brief_data["updated_by"] = "functional tests"
 
+  response = call_api(:post, "/briefs", payload: brief_data)
+  response.code.should be(201), response.body
+  brief = JSON.parse(response.body)["briefs"]
+  return brief
+end
+
+def publish_buyer_brief(brief_id)
+  publish_brief_data = {
+    updated_by: "functional tests",
+    briefs: {status: "live"}
+  }
+  response = call_api(:put, "/briefs/#{brief_id}/status", payload: publish_brief_data)
+  response.code.should be(200), response.body
+end
+
+def delete_all_draft_briefs (user_id)
   response = call_api(:get, "/briefs", params: {user_id: user_id})
-  # puts response.body
+
   JSON.parse(response.body)["briefs"].each do |brief|
-    puts brief["title"]
+    brief_id = brief["id"]
+
+    updated_by = {updated_by: "Functional tests"}
+
+    if brief["status"] != "live"
+      puts "deleting draft: #{brief_id}"
+      response = call_api(:delete, "/briefs/#{brief_id}", payload: updated_by)
+      response.code.should be(200), response.body
+    end
+  end
+end
+
+Given /^I have a '(.*)' brief$/ do |brief_state|
+  if not store.buyer_id
+    fail(ArgumentError.new('No buyer user found!!'))
   end
-  # if response.
-  # if response.code == 404
-  #   response = call_api(:post, "/users", payload: user_data)
-  #   response.code.should be(201), response.body
-  # end
-end
-#@wip-create a buyer brief
-Given /^I have a brief$/ do
-  create_buyer_brief("Individual Specialist-Brief deletion test","digital-outcomes-and-specialists","digital-specialists",10348)
+  brief = create_and_return_buyer_brief("Individual Specialist-Brief deletion test", "digital-outcomes-and-specialists", "digital-specialists", store.buyer_id)
```

Comment:
I was actually wondering if we could change this name to be a bit more general.
Would there be any problem changing "Individual Specialist-Brief deletion test" to something like "Individual Specialist Brief" or something?